patches/minecraft/net/minecraft/server/ReloadableServerResources.java.patch

```diff
@@ -44,3 +44,21 @@
     }
  
     private static <T> void m_206870_(RegistryAccess p_206871_, TagManager.LoadResult<T> p_206872_) {
+@@ -96,5 +_,17 @@
+          return p_206874_.getValue().m_6497_();
+       }));
+       p_206871_.m_175515_(resourcekey).m_203652_(map);
++   }
++
++   private final net.minecraftforge.common.crafting.conditions.ConditionContext context;
++
++   /**
++    * Exposes the currently staged TagManager wrapped in an IContext so that mods can use<br>
++    * tags in their custom reload listeners.<br>
++    * This is only useful for reload listeners who need the currently staged tag values.<br>
++    * @return A ConditionContext with access to the current TagManager.
++    */
++   public net.minecraftforge.common.crafting.conditions.ConditionContext getTagContext() {
```

Comment:
The whole reason the interface was named `IContext` was so we could add things besides tags to it in the future. Call this `getConditionContext`